### DISCUSSION TITLE
Manage more k8s resources

### DISCUSSION THREAD

**Discussion by cnCloXie** [kube-green/kube-green#367] [Ideas]
Hi team,

Is kube-green going to manage more k8s resources ? Such as 
- The deployment managed by HPA
- StatefulSet
- Cloud vendor resources, such as node group, rds, elasticache, kafka e.g

**Comment by davidebianchi** [comment]
Hi! Yes, kube-green will support more Kubernetes resources in the future. I'm working on it [here](https://github.com/kube-green/kube-green/pull/363).
At the moment, cloud vendor resources management is not in the kube-green scope, but maybe there are some use cases where it could be interesting. Have you some specific use cases to support those resources?

**Reply by cnCloXie** [reply]
Thank you for the information !
Maybe not accurate enough that I use the title "k8s resources", but we do consider to adopt crossplane which could manage cloud vendor resource vis kubernetes.

My use case is that, I need to shut down the databases, middlewares along with the Kubernetes runtimes, to treat them as a complete stack.
Such as the AWS support to temporally shut down RDS(database) for up to 7 days. Which allow me to write a golang script, shutting down the RDS when scale the pod & nodegroup down since already no applications need it. 
The same things for ElastiCache, MSK e.g resources Im working on.

**Reply by davidebianchi** [reply]
Oh, really interesting use case! Since crossplane uses Kubernetes resources, maybe with the change to support custom resources it could be possible to manage also the stop of those resources.
